DESCRIPTION

```diff
@@ -1,16 +1,16 @@
-Package: dashPlayer
-Title: Dash player component for videos
+Package: Dash Player
```

Comment:
This file is auto-generated and is used just by R, it shouldn't be edited manually.

If we want to support using this package with R, the right thing to do is to include it in the build command by adding a `--r-prefix` argument. I think we don't actually want a prefix for this one since `dash` is already part of the component name, so I'd suggest `--r-prefix ''`. The pattern we've been using recently is to then change the command name from `build:py` to `build:backends` (and update the full `build` command to point to the new name). And while you're at it, I think we can get rid of the `build:*-activated` commands, they're not really useful and it's easy for them to get out of sync.

Reply:
Thanks for the suggestion - fixed!